config/checkstyle/suppressions.xml

```diff
@@ -20,4 +20,5 @@
 <suppressions>
     <suppress checks="EqualsHashCode" files="shapes/*"/>
     <suppress checks="InnerAssignment" files="StringUtils"/>
+    <suppress checks="TypeName" files="AwsJson1_0Trait.java|AwsJson1_1Trait.java"/>
```

Comment:
These names are flagged as not meeting Java conventions, but we need to carry the version numbers in the generated class.